episodes/01-intro-to-r.Rmd

```diff
@@ -640,6 +640,62 @@ Now that we have learned how to write scripts, and the basics of R's data
+
+However, there are several other ways that people often get help when they are stuck with their R code.
+
+* Search the internet: paste the last line of your error message or “R” and a short description of what you want to do into your favorite search engine and you will usually find several examples where other people have encountered the same problem and came looking for help.
+  * Stack Overflow can be particularly helpful for this: answers to questions are presented as a ranked thread ordered according to how useful other users found them to be. You can search using the `[r]` tag. 
+  * **Take care**: copying and pasting code written by somebody else is risky unless you understand exactly what it is doing!
+* Ask somebody “in the real world”. If you have a colleague or friend with more expertise in R than you have, show them the problem you are having and ask them for help.
+* Sometimes, the act of articulating your question can help you to identify what is going wrong. This is known as [“rubber duck debugging”](https://en.wikipedia.org/wiki/Rubber_duck_debugging) among programmers.
+
+### Generative AI
+
+::::::::::::::::::::::::::::: instructor
+
+### Choose how to teach this section
+The section on generative AI is intended to be concise but Instructors may choose to devote more time to the topic in a workshop.
+Depending on your own level of experience and comfort with talking about and using these tools, you could choose to do any of the following:
+
+* Explain how large language models work and are trained, and/or the difference between generative AI, other forms of AI that currently exist, and the limits of what LLMs can do (e.g., they can't "reason").
+* Demonstrate how you recommend that learners use generative AI.
+* Discuss the ethical concerns listed below, as well as others that you are aware of, to help learners make an informed choice about whether or not to use generative AI tools.
+
+This is a fast-moving technology. 
+If you are preparing to teach this section and you feel it has become outdated, please open an issue on the lesson repository to let the Maintainers know and/or a pull request to suggest updates and improvements.
+
+::::::::::::::::::::::::::::::::::::::::
+
+It is increasingly common for people to use _generative AI_ chatbots such as ChatGPT to get help while coding.
+You will probably receive some useful guidance by presenting your error message to the chatbot and asking it what went wrong.
+
+However, the way this help is provided by the chatbot is different.
+Answers on Stack Overflow have (probably) been given by a human as a direct response to the question asked.
+But generative AI chatbots, which are based on advanced statistical modeld called Large Language Models (or LLMs), respond by generating the _most likely_ sequence of text that would follow the prompt they are given.
```

Comment:
@josenino95  Typo on line 676: `statistical modeld` should be `statistical models`